src/baldr/datetime.cc

```diff
@@ -126,10 +126,13 @@ std::string iso_date_time(const date::time_zone* time_zone) {
 
 // Get the seconds since epoch time is already adjusted based on TZ
 uint64_t seconds_since_epoch(const std::string& date_time, const date::time_zone* time_zone) {
-  if (date_time.empty() || !time_zone) {
+  if (date_time.empty()) {
     return 0;
   }
   const auto d = get_formatted_date(date_time);
+  if (!time_zone) {
+    return static_cast<uint64_t>(d.time_since_epoch().count());
+  }
```

Comment:
Ah right, almost forgot, this fixes #3825 at least partly. Still needs a bit more work in other functions here. this was just added so this PR works. The rest is still as broken as before when no timeinfo is present on the nodes.